src/firebase/firebase-config.ts

```diff
@@ -35,11 +38,11 @@ const firebaseConfig = {
 const app = initializeApp(firebaseConfig);
 const fbStorage = getStorage();
 const db = getFirestore(app);
-const auth = getAuth(app);
+// const auth = getAuth(app);
+const auth = initializeAuth(app, {
```

Comment:
Does it work now without breaking the app?

Reply:
IT works on mobile, but it's still an issue for web